src/wasm/syscalls_openmp.cpp

```diff
@@ -59,7 +72,7 @@ namespace wasm {
         Runtime::GCPointer<Runtime::Memory> &memoryPtr = getExecutingModule()->defaultMemory;
         
         // Spawn calls to the microtask in multiple threads
-        unsigned int numThreads = util::getUsableCores();
+        int numThreads = util::getUsableCores(); // or fetch local num_thread
```

Comment:
Yeah I think we can probably avoid repeated calls to get the number of threads that are scattered all over.